roles/openshift_cfme/files/miq-pv-backup-example.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: miq-pv03
+spec:
+  capacity:
+    storage: 15Gi
+  accessModes:
+  - ReadWriteOnce
+  nfs:
+    path: "/exports/miq-pv03"
+    server: "<your-nfs-host-here>"
```

Comment:
Can't this be populated from the existing nfs storage host information we're already gathering?

Reply:
oh snap. Those other example ones aren't even being used yet. I just bulk copied all of the ones from the upstream project.